code/modules/reagents/reagent_containers/patch.dm

```diff
@@ -37,4 +37,10 @@
 	name = "burn patch"
 	desc = "Helps with burn injuries."
 	list_reagents = list("silver_sulfadiazine" = 20)
-	icon_state = "bandaid_burn"
\ No newline at end of file
+	icon_state = "bandaid_burn"
+
+/obj/item/reagent_containers/pill/patch/synthflesh
+	name = "brute and burn patch"
+	desc = "Helps with brute and burn injuries."
+	list_reagents = list("synthflesh" = 15)
```

Comment:
This heals less than 40 damage whereas brute and burn patches heal over 100.

Reply:
Well I can up the dosage.